### DISCUSSION TITLE
v1/en/docs/image-processing/

### DISCUSSION THREAD

**Discussion by giscus** [razonyang/hugo-theme-bootstrap#895] [General]
# v1/en/docs/image-processing/

Guide to resize and align images

https://hbs.razonyang.com/v1/en/docs/image-processing/

**Comment by palladius** [comment]
Wow, this page is awesome! I just don't follow on one thing: How do i get my image nicely wrapped around the text on left or right? 
I was expecting the float would do, but it only gets one line.

I'm thinking something like this: https://www.geeksforgeeks.org/how-to-wrap-the-text-around-an-image-using-html-and-css/

Can you achieve it today? Apologies for the newbie question.

**Comment by razonyang** [comment]
It seems a bug, I'll create a patch later.

**Comment by razonyang** [comment]
Fixed by #897, you'll need to upgrade the theme to `v1.0.0-beta4.2`.

```markdown
{{/% clearfix %/}}
![Float Start](/featured-sample.webp#float-start) TEXTS AROUNDS THE IMAGE.
{{/% /clearfix %/}}
```

**Comment by liluzibird** [comment]
Hello, this is a nice guide. However, I am looking for the directory in which the images are stored. I made a .png, .jpg, and a .webp and put them in resources/_gen/images, in content/authors/user, and static/images. I can't seem to change the profile picture and logo, only the favicon so far.

**Reply by razonyang** [reply]
Hi, this guide is for Markdown content, for profile picture and logo, you should replace the `static/images/profile.webp` and `static/images/logo.webp`.
See https://github.com/razonyang/hugo-theme-bootstrap-skeleton/tree/main/static/images.

**Reply by liluzibird** [reply]
Yes, I tried putting my profile and logo in that directory, but they keep getting overwritten by the previous images.

**Reply by razonyang** [reply]
Is that a cache issue? Please restart Hugo server and force refresh on browser (CTRL+F5). If it's not, would you mind share your repo to me (you can invite me to your repo if it's private, and remove me later)?

**Reply by liluzibird** [reply]
For sure. The repo is https://github.com/uzerliluzibert/userliluzibert.github.io

**Reply by liluzibird** [reply]
Sorry, that was a typo, it's https://github.com/userliluzibert/userliluzibert.github.io

**Reply by razonyang** [reply]
It works great.

![image](https://github.com/razonyang/hugo-theme-bootstrap/assets/17720932/5bc9e5c2-8a10-4992-b9b4-e506fc74b93a)

**Reply by liluzibird** [reply]
interesting. I guess it just needed time to update

**Reply by liluzibird** [reply]
actually, where are you viewing this from? It doesn't show like that for me both on https:userliluzibert.github.io and both locally. Both of mine still haven't updated.

**Reply by liluzibird** [reply]
ahhhh, It works on a different browser

**Reply by razonyang** [reply]
Seems a cache issue, I can see the updated images on both of your live site and local site.

**Reply by liluzibird** [reply]
How careless of me. I could've saved so much time and headaches by just clearing the cache in firefox. Thanks bro.

**Reply by razonyang** [reply]
No problem, this may be fixed by #1012.
